plugins/helphub-read-time.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/*
+ * Plugin Name: HelpHub Read Time
+ * Description: Adds estimated reading time to a post using a simple formula.
+ * Version:     1.0.0
+ * Author:      justingreerbbi
+ * Author URI:  https://wordpress.org
+ * License:     GPL3
+ * License URI: https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+defined( 'ABSPATH' ) or die( 'No script kiddies please!' );
+
+// Adds the read time hook when saving a post
+add_action( 'save_post', 'hh_calulate_and_update_post_read_time', 10, 3 );
+
+/**
+ * Calculates the read time of a post when created or updated.
+ * @param  int 			$post_id Post id to calculate read time for.
+ * @param  object 	$post    Post object
+ * @param  boolean 	$update  Is this an update or new.
+ * @return void
+ */
+function hh_calulate_and_update_post_read_time( $post_id, $post, $update ) {
+
+	// No post revisions
+	if ( wp_is_post_revision( $post_id ) )
+		return;
+
+	// Get post types that need to have read time applied
+	$calculate_for_posts = apply_filters( 'read_time_types', array( 'post' ) );
+
+	// If the post type is not found then return
+	if ( ! in_array( $post->post_type, $calculate_for_posts ) )
+		return;
```

Comment:
Coding standards: Please use curly braces for all clauses, even if they're only one line.